src/app/components/post/post.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { PostService } from 'src/app/services/post.service';
+import { Post } from 'src/types/posts';
+
+@Component({
+    selector: 'app-post',
+    templateUrl: './post.component.html',
+    styleUrls: ['./post.component.scss'],
+})
+export class PostComponent implements OnInit {
+    @Input() postId: number;
+
+    post: Post;
+
+    postTitle: string;
```

Comment:
Too much space between. However, I think this may be related to bad eslint setup. If that's the case, it should be fixed with Jirinka's commits. We'll work on this some more later on

Reply:
Yes it was autofixed by eslint but I removed them